cegs_portal/search/templates/search/v1/experiment.html

```diff
@@ -431,6 +449,9 @@
 
 {% block inline_javascript %}
 <script>
+    const urlParams = new URLSearchParams(window.location.search);
+    const analysisId = urlParams.get('analysis');
```

Comment:
Why are you doing this instead of using the  analysis template value?

Reply:
The analysis template value is being used when we fetch url but the user can type in a different ID in params. We want that ID to be used to fill in for the analysis ID when calling vpot and qqplot graphs 
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/211fd2c0-cc7b-4a0b-ba1b-504e6efd7981)

Reply:
When or if they type "?analysis=DCPAN0000000002" in the url. The page isnt requested with ?analysis=DCPAN0000000002 when the user request an experiment page. 
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/d62aa390-22a1-49e4-b3c4-fbcbac71902b)

Reply:
When the user types it in the code uses it to fill in as the analysis ID

Reply:
typing `?analysis=DCPAN0000000002` into the URL doesn't do anything unless the user hits "enter", reloading the page. Like, if the page is loaded and the user, after the page is loaded, types `?analysis=DCPAN0000000002` into the URL, nothing will happen. The user has to hit "enter", which will do a full load of the page, with a call to the backend.

Reply:
Thats the same thing for assembly. The user has to hit enter as well when the type in ref genome.
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/21034a55-342d-4f1e-9f22-dcdbab41bf27)

Reply:
no, because `URLSearchParams` isn't used at all in the dna_feature.html template.